### PR TITLE
gluon-status-page: add role to status-page

### PR DESCRIPTION
**Pull request by AiyionPrime**
Implements the next open crumb of #1415.

### CODE REVIEW

File: package/gluon-status-page/i18n/de.po [hunk 1]
```diff
@@ -97,6 +97,9 @@ msgstr "RAM"
 msgid "Received"
 msgstr "Empfangen"
 
+msgid "Received"
```
Comment (lemoer) [inline package/gluon-status-page/i18n/de.po:100]:
```suggestion
msgid "Received"
```

This should be 'Role'.

Comment (AiyionPrime) [reply package/gluon-status-page/i18n/de.po:100]: indeed thanks.

**Review by mweinelt — APPROVED** [review]
LGTM.

**Comment by lemoer** [comment]
LGTM.

**Review by lemoer — APPROVED** [review]

**Comment by lemoer** [comment]
Let's merge this?

**Comment by NeoRaider** [comment]
I think this should resolve the human-readable role name via i18n instead of using the internal ID.

**Comment by lemoer** [comment]
For reference: Neoraider is talking about this: https://github.com/freifunk-gluon/gluon/blame/a2be901803469d378d6469551a65c20f744aefff/docs/releases/v2015.1.rst#L189-L200

**Comment by AiyionPrime** [comment]
Thanks @lemoer, though the referenced code is outdated;
the translations are now in `gluon-web-node-role:role:<rolename>`

**Comment by AiyionPrime** [comment]
> I think this should resolve the human-readable role name via i18n instead of using the internal ID.

@NeoRaider The last commit adds your requested translation, though I do not know, whether there's a better syntax inside of views.

**Comment by NeoRaider** [comment]
> > I think this should resolve the human-readable role name via i18n instead of using the internal ID.
> 
> @NeoRaider The last commit adds your requested translation, though I do not know, whether there's a better syntax inside of views.

I think it's fine, the special i18n view syntax isn't usable here because the site is a foreign i18n package.

**Comment by AiyionPrime** [comment]
Is there anything left to do on my end, then?

**Review by mweinelt — APPROVED** [review]

**Pull request merged**